prime-dotnet-webapi/Models/TermsOfAccessLicenseClassClause.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("TermsOfAccessLicenseClassClauseXref")]
```

Comment:
We are not putting "Xref" in the names of our Xref tables. I'm sure there is a coffee shop argument to be had here.

Reply:
Sounds good, removed.  Nuked, paved, and pushed